src/app/api/create-meme/route.ts

```diff
@@ -1,7 +1,7 @@
+import { Buffer } from "buffer";
 import { NextResponse } from "next/server";
 
 export async function GET(request: Request) {
```

Comment:
This should be a NextRequest type.  Also this should always return a NextResponse with an error object rather than throwing.